libdebug/debugger/internal_debugger.py

```diff
@@ -279,6 +291,16 @@ def detach(self: InternalDebugger) -> None:
 
         self._join_and_check_status()
 
+    @property
+    def arch(self: InternalDebugger) -> str:
+        """The architecture of the debugged process."""
+        return self._arch
```

Comment:
`_arch` is not defined, but `arch`

Reply:
I don't understand what you mean. `arch` is declared and implemented, `_arch` is defined in the constructor and accessed here.

Reply:
I mean the definition on top of the file, where types and descriptions are provided. `_arch` is never defined like in the other files of libdebug where the private attributes are defined (and not the public properties).